<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>history</class>
 <widget class="QWidget" name="history">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="back">
       <property name="minimumSize">
        <size>
         <width>31</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>31</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>â¬…</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Browsing History</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="image1">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="name1">
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Name, Distance</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="viewprofile_1">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 40px;
    background-color: #fdd5b1;
}</string>
       </property>
       <property name="text">
        <string>VIEW</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="image2">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="name2">
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Name, Distance</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="viewprofile_2">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 40px;
    background-color: #fdd5b1;
}</string>
       </property>
       <property name="text">
        <string>VIEW</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="image3">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="name3">
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Name, Distance</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="viewprofile_3">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 40px;
    background-color: #fdd5b1;
}</string>
       </property>
       <property name="text">
        <string>VIEW</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="prev">
       <property name="font">
        <font>
         <family>Manjari</family>
        </font>
       </property>
       <property name="text">
        <string>Prev</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="page">
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Page 1 of X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="next">
       <property name="font">
        <font>
         <family>Manjari</family>
        </font>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
